test/performance/config/performance-reporter.ts

```diff
@@ -94,6 +96,9 @@ export function curateResults(
 		type: average( results.type ),
 		minType: minimum( results.type ),
 		maxType: maximum( results.type ),
+		typeWithTopToolbar: average( results.typeWithTopToolbar ),
+		minWithTopToolbarType: minimum( results.typeWithTopToolbar ),
+		maxWithTopToolbarType: maximum( results.typeWithTopToolbar ),
```

Comment:
These aren't necessary at this point. We didn't add them with the recently added `typeWithToolbar` as well.